Translation corrections (#464)
07th-mod/higurashi-console-arcs: Update/omot_015.txt

			NULL, "\"Yes, that's right.", Line_WaitForInput);
	ModPlayVoiceLS(4, 9, "ps3/s14/09/210900047", 256, TRUE);
	OutputLine(NULL, "残念ながら人間として命を永らえることが出来なかった人が、その力を神へとささげることでシャングリラへの道が開かれる」",

> 			NULL, " Unfortunately, those who have failed to live a full life as humans can offer their power to the gods and open the way to Shangri-La.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " Those who have unfortunately failed to live a full life as humans can still offer their power to the gods and open the way to Shangri-La.\"", Line_Normal);
```

				NULL, "\"Nfu fu fu, I've heard a lot about you~!", Line_WaitForInput);
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1011", 256, TRUE);
		OutputLine(NULL, "　…それにしても、やっぱり聞くと見るとでは、全然違いますねぇ！」",

> 				NULL, " ...Still finally getting the chance to see you is completely different from hearing about you!\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " ...Still, finally getting the chance to see you is completely different from hearing about you!\"", Line_Normal);
```

				NULL, "\"Na ha ha ha!", Line_WaitForInput);
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1015", 256, TRUE);
		OutputLine(NULL, "　こりゃ心外な、私は本心からそう言ったんですがねぇ」",
				NULL, " Why, I'm hurt. I meant that sincerely.\"", Line_Normal);
		ClearMessage();

	//大石rvS14/11/VT4Bc_oishi1016.「いやいや赤坂さん、実にあなたが羨ましい！kvS14/11/VT4Bc_oishi1017.　こんな綺麗な方の旦那さんがあなたじゃなかったら私、今晩神社にこもって五寸釘を藁人形に打ち付けていますって！kvS14/11/VT4Bc_oishi1018.　んッふっふっふっ…！」
		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#a59da9>大石</color>", NULL, "<color=#a59da9>Ooishi</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1016", 256, TRUE);
		OutputLine(NULL, "「いやいや赤坂さん、実にあなたが羨ましい！",
				NULL, "\"I must say, Akasaka-san, I'm actually quite jealous!", Line_WaitForInput);
		ModPlayVoiceLS(4, 11, "ps3/s14/11/vt4bc_oishi1017", 256, TRUE);
		OutputLine(NULL, "　こんな綺麗な方の旦那さんがあなたじゃなかったら私、今晩神社にこもって五寸釘を藁人形に打ち付けていますって！",

> 				NULL, " If this pretty lady's husband wasn't you, I'd be down at the shrine tonight nailing up straw dolls!", Line_WaitForInput);

Reviewer: ```suggestion
				NULL, " If this pretty lady's husband was anyone but you, I'd be down at the shrine tonight nailing up straw dolls!", Line_WaitForInput);
```

				NULL, "\"I'm sure you and Ooishi-san have plenty of stories to share, right?", Line_WaitForInput);
		ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bc_yukie1014", 256, TRUE);
		OutputLine(NULL, "　めったにお会いできない大切な方なんだから、良いじゃありませんか。",
				NULL, " He's a good friend you rarely get to see, so it's okay, isn't it?", GetGlobalFlag(GLinemodeSp));
		if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>雪絵</color>", NULL, "<color=#f5e6d3>Yukie</color>", NULL, Line_ContinueAfterTyping); }
		ModPlayVoiceLS(4, 0, "ps3/s14/00/vt4bc_yukie1015", 256, TRUE);
		OutputLine(NULL, "私と美雪は温泉地で、ゆっくりしていますわ」",

> 				NULL, " Miyuki and I will head to the hot spring and take our time there.\"", Line_Normal);

Reviewer: ```suggestion
				NULL, " Miyuki and I will head to the hot spring and relax.\"", Line_Normal);
```
